packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -606,7 +606,9 @@ class StackViewport extends Viewport implements IStackViewport {
     this.hasPixelSpacing = scale > 0 || imagePlaneModule.rowPixelSpacing > 0;
     imagePlaneModule.calibration = calibration;
 
-    if (!isUpdated) return imagePlaneModule;
+    if (!isUpdated) {
```

Comment:
This was unnecessary - please revert it.  If the entire if/then is on a single line, then it should stay that way.